tests/plugin.test.js

```diff
@@ -214,8 +214,8 @@ describe("LegacyESLint", () => {
 
 			assert.strictEqual(results.length, 1);
 			assert.strictEqual(results[0].messages.length, 1);
-			assert.notProperty(results[0].messages[0], "endLine");
-			assert.notProperty(results[0].messages[0], "endColumn");
+			assert.ok(!Object.hasOwn(results[0].messages[0], "endLine"));
+			assert.ok(!Object.hasOwn(results[0].messages[0], "endColumn"));
```

Comment:
It looks like Chai's `assert.notProperty` also checks non-own properties, for example `assert.notProperty({}, "constructor")` is an assertion error. I think we could use the `in` operator here:

```suggestion
			assert.ok(!("endLine" in results[0].messages[0]));
			assert.ok(!("endColumn" in results[0].messages[0]));
```